tracing/opencensus_test.go

```diff
@@ -47,3 +48,43 @@ func TestOpenCensusTracerGlobalLifecycle(t *testing.T) {
 	}
 	otherOCT.Finish()
 }
+
+func TestOpenCensusTrace_ApplyConfig_FailingConfigOption(t *testing.T) {
+	coErr := errors.New("configOption error")
+	oct := NewOpenCensusTracer(func(c *config.Config) error {
+		if c != nil {
+			return coErr
+		}
+		return nil
+	})
+	if err := oct.ApplyConfig(&config.Config{}); err != coErr {
+		t.Errorf("Expected error not seen. Expected %q. Actual %q", coErr, err)
```

Comment:
Here and elsewhere: "got before want", e.g. `error = %v, want: %v`.

Reply:
Done.